Sources/DiscordKitCore/REST/APIRequest.swift

```diff
@@ -174,6 +174,23 @@ public extension DiscordREST {
         )
     }
 
+    /// Make a `POST` request to the Discord REST APIfor endpoints
```

Comment:
Fixes a little typo :)

```suggestion
    /// Make a `POST` request to the Discord REST API for endpoints
```